hal/lib/athena/handles/IndexedHandleResource.h

```diff
@@ -18,8 +18,8 @@
 
 namespace hal {
 
-constexpr int32_t IndexedResourceIndexOutOfRange = -1;
-constexpr int32_t IndexedResourceNotAllocated = -2;
+constexpr int32_t HandleIndexOutOfRange = -1;
+constexpr int32_t HandleAlreadyAllocated = -2;
```

Comment:
Hmm, since we're setting status values to these shouldn't they be part of the public API?  We should also consider making status values unique (not function dependent) across the API, but that can be left to a later commit.